config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -103,7 +103,7 @@ periodics:
       - --provider=gce
       - --metadata-sources=cl2-metadata.json
       - --env=CL2_LOAD_TEST_THROUGHPUT=50
-      - --env=CL2_DELETE_TEST_THROUGHPUT=30
+      - --env=CL2_DELETE_TEST_THROUGHPUT=60
```

Comment:
How about aligning it for the moment with the creation throughput and setting to 50?

And potentially increasing both further in the subsequent change?

Reply:
I was thinking about this for a while. I don't have a strong opinion on this. I decided to use 60 as it's faster, we know that it will work (this is the value I used for manual test) and ensures stronger property we can use later internally.

Reply:
If you don't feel very strongly, I personally prefer consistency and setting it to 50.
Yes - it will be slightly slower, but the difference is not that significant. If we really want to speed it up, let's do that in a followup and bump both values.

Reply:
Agreed. Changed to 50.